App.tsx

```diff
@@ -18,28 +18,43 @@ export default function App() {
     JetBrainsMono_400Regular,
     JetBrainsMono_700Bold,
   });
+  const [appIsReady, setAppIsReady] = useState(false);
 
   useEffect(() => {
     async function prepare() {
-      await SplashScreen.preventAutoHideAsync();
+      try {
+        await SplashScreen.preventAutoHideAsync();
+        await new Promise((resolve) => setTimeout(resolve, 2000));
+      } catch (e) {
+        console.warn(e);
+      } finally {
+        setAppIsReady(true);
+      }
```

Comment:
Can this fail? Do we need a try/catch block here?

Reply:
I think it's better if we let it for safety reasons